source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -599,6 +601,15 @@ int StreamHandleWrapper::luaImportPublicKey(lua_State* state) {
   return 1;
 }
 
+int StreamHandleWrapper::luaBase64Escape(lua_State* state) {
+  // Get input string.
+  absl::string_view input = luaL_checkstring(state, 2);
+  auto output = absl::Base64Escape(input);
```

Comment:
If absl has base64 support can we get rid of base64.h/cc and/or pass through to absl? (Fine as follow up). cc @lizan @jmarantz

Reply:
Yeah, might be possible to pass through to absl, seems it can do base64 / hex. One thing it doesn't do for base64 is encoding our buffer without linearizing.

Reply:
OK I will file a follow up issue here.